### PR TITLE
Antigene Test Result Notification (EXPOSUREAPP-6039)

### PR DESCRIPTION
**Pull request by chris-cwa**
new notification service for rat tests - code review only, this can be tested after other PRs are merged

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/notification/TestResultAvailableNotificationService.kt [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
 import de.rki.coronawarnapp.util.notifications.setContentTextExpandable
 import kotlinx.coroutines.flow.first
 import timber.log.Timber
-import javax.inject.Inject
 import javax.inject.Provider
 
-class TestResultAvailableNotificationService @Inject constructor(
-    @AppContext private val context: Context,
+open class TestResultAvailableNotificationService(
+    private val context: Context,
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/notification/TestResultAvailableNotificationService.kt:17]: @AppContext  ?

**Review by mtwalli — COMMENTED** [review]
LGTM , please fix the test? then I will approve

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/notification/TestResultAvailableNotificationService.kt (hunk 1 above)
Comment (chris-cwa) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/notification/TestResultAvailableNotificationService.kt:17]: this class won't be injected, only subclasses of it, so no injection annotations needed, i guess?

**Review by mtwalli — APPROVED** [review]
Thanks

**Review by d4rken — APPROVED** [review]
lgtm

**Pull request merged**